recipe/meta.yaml

```diff
@@ -34,8 +32,15 @@ requirements:
     - pyyaml >=3.10
 
 test:
+  source_files:
+    - setup.cfg
+    - tests/
   requires:
     - pip
+    - pytest
+    - pytest-timeout
+    - pytest-cov
+    - flaky
   imports:
     - watchdog
     - watchdog.observers
```

Comment:
Don't think it's a blocker, but I'm not seeing `watchdog.observers.fsevents`. Was this possibly replaced with `_watchdog_fsevents`?

Reply:
Also, why the first underscore in `_watchdog_fsevents`?

Reply:
`watchdog.observers.fsevents` is https://github.com/gorakhargosh/watchdog/blob/v4.0.1/src/watchdog/observers/fsevents.py

Reply:
Seems` _watchdog_fsevents` for `osx` is https://github.com/gorakhargosh/watchdog/blob/v4.0.1/setup.py#L44